README.md

```diff
@@ -12,6 +12,12 @@ These reports are incubated by the [Solid Notifications Panel](https://github.co
 * Editors Draft:
   * [Solid Notifications Protocol](https://solid.github.io/notifications/protocol)
 
+### Quick Protocol
+* Latest Published Version:
+  * [Solid Quick Notifications Protocol](https://solidproject.org/TR/quick-notifications-protocol)
```

Comment:
404 I would suggest to add that link once it gets published

Reply:
My intention was that when the PR gets accepted, it will be created.

Happy to comment it out if it is a problem...